server/etcdmain/grpc_proxy.go

```diff
@@ -19,6 +19,7 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"google.golang.org/grpc/metadata"
```

Comment:
This import is mislocated according to the static check. It should be grouped in the next section.

Reply:
Thanks for your reviewing, I'll fix it soon.